clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/arrays/ArraysMetadata.scala

```diff
@@ -28,6 +28,9 @@ case class ArraysMetadata(
   pipelineVersion: Option[Symbol] = None,
   project: Option[String] = None,
   redIdatPath: Option[URI] = None,
+  refFastaPath: Option[URI] = None,
+  refFastaIndexPath: Option[URI] = None,
+  refDictPath: Option[URI] = None,
```

Comment:
do we care about validating that the file prefixes are the same?

Reply:
In theory, what files we use as references could change. What is the relationship of the dict file to the fasta file? Do we do this file prefix validation on other files that have indexes?

Reply:
I think typically we have only supplied the fasta file and deduced the other file names from that. I'm not saying that is ideal either but is less likely to trip up htsjdk.

Reply:
I could either do that validation here or change it (here, in picard-private, and in dsde-pipelines) to accept a file prefix and append the extensions.

Reply:
Personally I prefer having them all defined instead of deriving them. There are likely tools that don't care about prefixes. Even the validation feels like it should only be a warning to me, but I wouldn't be surprised if this validation happens upstream of calls to clio anyway. I think it is fine as is.